release_creation/bundle/download_no_deps.sh

```diff
@@ -5,6 +5,9 @@ platform=$2
 requirements_file=$3
 staging="download-no-deps-staging"
 
+rm -rf $staging
```

Comment:
Make sure that the staging directory always starts fresh, so that nothing "carries over" from previous runs. We do this at earlier steps in the process.